packages/shared/src/types/abTests/banner.ts

```diff
@@ -75,13 +66,22 @@ export interface BannerTestSelection {
     targetingAbTest?: TargetingAbTest;
 }
 
-// Models for the config from the RRCP
-export type BannerVariantFromTool = Omit<
-    BannerVariant,
-    'modulePathBuilder' | 'componentType' | 'products'
->;
-export type BannerTestFromTool = Omit<BannerTest, 'bannerChannel' | 'isHardcoded'> & {
-    variants: BannerVariantFromTool[];
-};
+export interface BannerTest extends BannerTestFromTool {
+    bannerChannel: BannerChannel;
+    isHardcoded: boolean;
+    canRun?: CanRun;
+    purchaseInfo?: PurchaseInfoTest;
+    variants: BannerVariant[];
+}
+
+export const bannerTestFromToolSchema = testSchema.extend({
+    userCohort: userCohortSchema,
+    locations: z.array(countryGroupIdSchema),
+    contextTargeting: pageContextTargetingSchema,
+    variants: z.array(bannerVariantFromToolSchema),
+    articlesViewedSettings: articlesViewedSettingsSchema.optional(),
+});
+
+export type BannerTestFromTool = z.infer<typeof bannerTestFromToolSchema>;
```

Comment:
we decided to name this with `FromTool` as opposed to `DB` we used for the other schemas? What do you think @tomrf1 ? We can go back and make this consistent

Reply:
As long as it's consistent across all then I don't mind!